data/formats-data.js

```diff
@@ -6113,6 +6132,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	dewpider: {
+		randomBattleMoves: ["scald", "leechlife", "infestation", "stickyweb", "rest", "sleeptalk"],
```

Comment:
remove infestation and scald, add liquidation - dewpider's water bubble liquidation is a force to be reckoned with

Reply:
sure to liq but those other two def still seem useful to defensive sets